### PR TITLE
Update article on the GitHub CLI

### LINKED ISSUE #4845: Update article on the GitHub CLI
### What article on docs.github.com is affected?

"[GitHub CLI](https://docs.github.com/en/github/getting-started-with-github/github-cli)"

### What part(s) of the article would you like to see updated?

- Improve the article intro to make it friendlier to new users and more useful for all readers
- In the “About GitHub CLI” section, make the list of features more general and evergreen, so it doesn’t become out of date when more features are shipped to the CLI. For example, before the list, change the wording to something like “The GitHub CLI includes features like…”

### Additional information

This issue is for a special project and not currently available for contributors to work on.

### PR DESCRIPTION
**Pull request by thepushkarp**
### Why:

Closes #4845

### What's being changed:

- Change 'computer' to 'terminal' to make intended use of the tool more specific (computer is a more general term here)
- Change the wordings in the 'About GitHub CLI' section to make it more general and add more features in the list

### Check off the following:

- [x] I have reviewed my changes in staging (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (eg. a "before and after video")

### CODE REVIEW

**Comment by janiceilene** [comment]
🎉 Thanks so much for opening a PR for this fix @thepushkarp! I'll triage this for review 💖

File: content/github/getting-started-with-github/github-cli.md [hunk 1]
```diff
@@ -11,14 +11,17 @@ topics:
 
 ### About {% data variables.product.prodname_cli %}
 
-{% data variables.product.prodname_cli %} is an open source tool for using {% data variables.product.product_name %} from your computer's command line. When you're working from the command line, you can use the {% data variables.product.prodname_cli %} to save time and avoid switching context.
+{% data variables.product.prodname_cli %} is an free and open source tool for using {% data variables.product.product_name %} from your computer's command line. When you're working from the command line, you can use the {% data variables.product.prodname_cli %} to save time and avoid switching context.
```
Comment (myarb) [inline content/github/getting-started-with-github/github-cli.md:14]: Some of the users of GitHub CLI are paying customers, so I've suggested removing this word ✨ 
```suggestion
{% data variables.product.prodname_cli %} is an open source tool for using {% data variables.product.product_name %} from your computer's command line. When you're working from the command line, you can use the {% data variables.product.prodname_cli %} to save time and avoid switching context.
```

Comment (myarb) [reply content/github/getting-started-with-github/github-cli.md:14]: In general, when updating conceptual content and intros like this, we try to focus on telling users the "why" behind the feature. 

For GitHub CLI, here is the high-level value we'd like to tell users about: GitHub CLI allows you to interact with GitHub features without requiring you to switch contexts out of a command line environment. In other words, GitHub CLI allows you to interact with GitHub functionality and features (from PRs, issues, to GitHub Actions) where you are already doing your work.

**Review by myarb — COMMENTED** [review]
Thanks so much for opening this PR ✨ In general, when updating conceptual content and intros like this, we try to focus on telling users the "why" behind the feature. I included some guidance in the review comments to consider when reworking this intro. Can you try writing this intro again and ping Janice when it's done? 💖

File: content/github/getting-started-with-github/github-cli.md (hunk 1 above)
Comment (thepushkarp) [reply content/github/getting-started-with-github/github-cli.md]: Thanks @myarb for the insightful comments. I've changed the intro according to what you said. Does this look appropriate?

Comment (MarkPhillips00763) [reply content/github/getting-started-with-github/github-cli.md]: This is me MarkPhillips who this one MarkPhillips00763@yahoo.com

**Comment by thepushkarp** [comment]
@janiceilene thanks for the review. I've added the changes as suggested by @myarb

**Comment by janiceilene** [comment]
Awesome, thanks for making updates @thepushkarp! I'll triage this for the team to review ⚡

File: content/github/getting-started-with-github/github-cli.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: GitHub CLI
-intro: 'The {% data variables.product.prodname_cli %} is a command-line tool for using {% data variables.product.product_name %} features on your computer.'
+intro: '{% data variables.product.prodname_cli %} is a command-line tool that brings pull requests, issues, workflows, and other {% data variables.product.product_name %} features to the terminal, without requiring you to switch contexts out of your work'
```
Comment (lecoursen) [inline content/github/getting-started-with-github/github-cli.md:3]:
```suggestion
intro: '{% data variables.product.prodname_cli %} is a command-line tool that brings pull requests, issues, workflows, and other {% data variables.product.product_name %} features to your terminal, so you can do all your work in one place.'
```

File: content/github/getting-started-with-github/github-cli.md [hunk 3]
```diff
@@ -13,12 +13,15 @@ topics:
...
+The {% data variables.product.prodname_cli %} includes {% data variables.product.prodname_dotcom %} features such as:
 
 - View, create, clone, and fork repositories
 - Create, close, and list issues and pull requests
 - Review, diff, and merge pull requests
-- Create, edit, list, and view gists
+- Create, edit, list, view, and delete gists
+- Create, list, view, and delete releases
+- Run, list, view, and list workflows
+- List, view, watch, and rerun workflow runs
```
Comment (lecoursen) [inline content/github/getting-started-with-github/github-cli.md:24]: I'm reordering these to go from more frequently used to less frequently used features. Also, IMO, "run... workflows" already covers the final bullet point. Let's condense them into one to reduce the cognitive effort required to read the article.

```suggestion
- Run, view, and list workflows
- Create, list, view, and delete releases
- Create, edit, list, view, and delete gists
```

**Review by lecoursen — COMMENTED** [review]
Thanks @thepushkarp, this is great! I had a few more small tweaks, which I'll apply now and then merge! 🎉

File: content/github/getting-started-with-github/github-cli.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: GitHub CLI
-intro: 'The {% data variables.product.prodname_cli %} is a command-line tool for using {% data variables.product.product_name %} features on your computer.'
+intro: '{% data variables.product.prodname_cli %} is a command-line tool that brings pull requests, issues, workflows, and other {% data variables.product.product_name %} features to your terminal, so you can do all your work in one place.'
```
Comment (lecoursen) [inline content/github/getting-started-with-github/github-cli.md:3]:
```suggestion
intro: '{% data variables.product.prodname_cli %} is a command-line tool that brings pull requests, issues, {% data variables.product.prodname_actions %}, and other {% data variables.product.product_name %} features to your terminal, so you can do all your work in one place.'
```

**Review by lecoursen — APPROVED** [review]

**Comment by thepushkarp** [comment]
Thank you so much @lecoursen! I checked the later commits and understood what could have been improved upon :smile:. I'll keep these little things in mind while working on the next issue. It was a great experience working on this one!

**Comment by CREATIONuri** [comment]
https://support.google.com/recaptcha/?hl=en


Get Outlook for ***@***.***> Cardgen.io<https://example.css/wordpress/freedomfinancial.ifo>

**Pull request merged**